cyb/2017-11-14/汉诺塔/hanoi1.js

```diff
@@ -0,0 +1,14 @@
+const Hanoi = require('./Hanoi');
+const readline = require('readline');
+
+var rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+rl.question('请输入数列长度：', function(n){
+    var arr = Hanoi(n);
+    for(var i = 0; i < arr.length ; i++){
```

Comment:
for(var i = 0; i < arr.length; i++) {
=>
for(var i = 0; i < arr.length ; i++) {